frontend/packages/insights-plugin/locales/ja/insights-plugin.json

```diff
@@ -1,15 +1,16 @@
 {
-  "low": "低 (low)",
-  "moderate": "中程度 (moderate)",
-  "important": "重要 (important)",
-  "critical": "重大 (critical)",
+  "Low": "低 (Low)",
+  "Moderate": "中程度 (Moderate)",
+  "Important": "重要 (Important)",
+  "Critical": "重大 (Critical)",
   "Temporary unavailable.": "一時的に利用不可です。",
   "Disabled or waiting for results.": "無効にされているか、または結果を待機しています。",
   "Total issue_0": "全体の問題",
   "Total issue_1": "全体の問題",
   "Total Risk": "全体リスク",
   "Fixable issues": "修正可能な問題",
   "View all in OpenShift Cluster Manager": "OpenShift Cluster Manager ですべてを表示する",
+  "Go to OpenShift Cluster Manager": "Go to OpenShift Cluster Manager",
```

Comment:
@rebeccaalpert, @spadgett, could you please help me with that and explain the algorithm of updating/adding i18n keys? Who we need to ask to translate this? How much time will it take? I prefer to finish translation in the scope of a separate patch ideally.

Reply:
Hi @gkarat - Thanks for tagging us here. You shouldn't ever update these files in the ja, zh, or ko folders. Sam and I take care of arranging translation for the whole console (including packages) right now, so there's nothing you need to do here beyond reverting the changes you've made to these files. We send over files about once a week now for translation.